rollup.config.js

```diff
@@ -0,0 +1,53 @@
+import commonjs from 'rollup-plugin-commonjs'
+import { terser } from 'rollup-plugin-terser'
+import globals from 'rollup-plugin-node-globals'
+import resolve from 'rollup-plugin-node-resolve'
+import json from '@rollup/plugin-json'
+
+import typescript from 'rollup-plugin-typescript2'
+
+const PROD = !!process.env.CI
+
+console.log(`production: ${PROD}`)
+
+const plugins = [
+  json({
+    compact: true
+  }),
+  typescript({
+    verbosity: 2,
+    clean: true,    useTsconfigDeclarationDir: true
+
+  }),
+  resolve({
+    browser: true,
+    preferBuiltins: false
+  }),
+  commonjs({
+    ignoreGlobal: true,
+    include: [/node_modules/],
+    namedExports: {
+      '@dcl/schemas': ['ChainId']
```

Comment:
This is required for each thing we require from dcl/schemas, right? I'm just a bit scared that the config is hidden and someone else updating the lib might break the build inadvertently

Reply:
This is only required for libraries we want to use with the SDK. It bundles @dcl/schemas with the library so it's not required as a dependency anymore (so we don't need to also build @dcl/schemas with the same method to work in the SDK)